### ISSUE TITLE
Disable/enabled buttons that is available

### ISSUE THREAD

**Issue by andwah** [Starcounter/Warehouse#45]
Disable buttons that is not needed or should not be pressed.

Example:

- Disable the `Update` button if there is nothing to update
- Disable the `Update` button if there is any validation errors on the form.

**Comment by miyconst** [comment]
This is a good and bad approach at same time.

The problem is that we update text values on change, and change happens when user loses focus of the input. So, if I want to modify only one value, then I have to set focus to the field, change value, click somewhere else, then button activates, press the button.

**Comment by andwah** [comment]
So i guess the solution is to trigger on `key-press` for all fields on on `focus lost`.

**Comment by miyconst** [comment]
Also not the silver bullet. This will trigger patches to/from server on each keypress, which results in higher server load and traffic usage.

**Comment by andwah** [comment]
True, but we will have a happier user. I think the higher load and usage is worth it!
